### PR TITLE
changed names related to renaming in sdk/protocol

### PR DESCRIPTION
**Pull request by mcarmonaa**

### CODE REVIEW

**Comment by mcarmonaa** [comment]
Travis is failing because of the current sdk/master doesn't contain the renaming changes yet

File: cmd/bblfsh/client.go [hunk 1]
```diff
@@ -53,8 +52,8 @@ func (c *clientCmd) Execute(args []string) error {
 		encoding = protocol.UTF8
 	}
 
-	req := &protocol.ParseUASTRequest{
-		Filename: filepath.Base(c.Args.File),
+	req := &protocol.ParseRequest{
+		Filename: c.Args.File,
```
Comment (abeaumont) [inline cmd/bblfsh/client.go]: If I understood it correctly, this line should not change

**Comment by abeaumont** [comment]
@mcarmonaa `glide.lock` should be updated to take the just merged `sdk` version with API changes

File: glide.lock [hunk 2]
```diff
@@ -12,7 +12,7 @@ imports:
   - uast
   - uast/ann
 - name: github.com/BurntSushi/toml
-  version: b26d9c308763d68093482582cea63d69be07a0f0
+  version: a368813c5e648fee92e5f6c30e3944ff9d5e8895
```
Comment (juanjux) [inline glide.lock]: Mmm, I'm not sure we should update all dependencies to their `~master` every time we update the SDK one. We probably should just update to the latest releases, trough that could depend on every dependency.

Comment (abeaumont) [inline glide.lock]: Agree. This should be fixed with proper versioning.

**Comment by abeaumont** [comment]
@mcarmonaa travis is failing due to `enry`, please take a look at it.

File: glide.lock (hunk 2 above)
Comment (mcarmonaa) [inline glide.lock]: note that set a version to master downloads the latest release, not the real last commit in master :S

**Pull request merged**